admin/includes/modules/document_product/collect_info_metatags.php

```diff
@@ -31,7 +31,7 @@
                                                FROM " . TABLE_META_TAGS_PRODUCTS_DESCRIPTION . "
                                                WHERE products_id = " . (int)$_GET['pID']);
   if ($check_meta_tags_description->RecordCount() <= 0) {
-    $product = $db->Execute("SELECT pd.products_name, p.products_model, p.products_price_sorter,
+      $product = $db->Execute("SELECT pd.products_name, p.products_model, p.products_price_sorter,
```

Comment:
Please remove these extra spaces in all three files where you have made the same change.

Reply:
PSR-12 says 4 spaces for ident.
And tangentially:https://github.com/zencart/zencart/pull/5073

Reply:
Yes but the rest of the file is 2.  Please remove and I will merge.

Reply:
all two.